dsc/src/subcommand.rs

```diff
@@ -282,11 +283,11 @@ pub fn config(subcommand: &ConfigSubCommand, parameters: &Option<String>, mounte
                     exit(EXIT_DSC_ERROR);
                 }
             };
-            (new_parameters, config_json)
+            (new_parameters, config_json, output_format)
         }
     };
 
-    let mut configurator = match Configurator::new(&json_string) {
+    let mut configurator = match Configurator::new(&json_string, output_format.as_ref()) {
```

Comment:
I don't like having `output_format` as a parameter to the constructor.  Would prefer adding a `set_output_format()` function instead.  Alternatively since this is something used all over the place, maybe put into thread local storage and just retrieve from there instead of passing it around particularly since it shouldn't change during runtime.